main.rb

```diff
@@ -0,0 +1,101 @@
+require './app'
+
+@app = App.new
+
+OPTIONS = {
+  1 => :list_all_books,
+  2 => :list_all_people,
+  3 => :create_person,
+  4 => :create_book,
+  5 => :create_rental,
+  6 => :list_rentals_for_person,
+  7 => :exit_program
+}.freeze
+
+def main
+  display_menu
+
+  loop do
+    choice = prompt_choice
+    handle_choice(choice)
+    break if choice == 7
+  end
+end
+
+def display_menu
+  puts 'Please select an option:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
```

Comment:
- [ ] If I chose any option from list and want to go back to the menu it is only showing me `Enter your choice (1-7): `. This is not a good practice. If I create many books or many users the list goes long and user might get lost which option is which. Kindly display list again alongside `Enter your choice (1-7): ` after completing mission in option.


<img width="590" alt="Screenshot 2023-06-22 at 22 28 17" src="https://github.com/danielamoreno699/school-library/assets/69011963/54ffa377-e3db-4264-b52c-c67d6762fa98">